Sherlock/test.cc

```diff
@@ -750,13 +767,19 @@ TEST(Sherlock, HTTPSubscriptionCanBeTerminated) {
 const std::string sherlock_golden_data =
     "{\"index\":0,\"us\":100}\t{\"x\":1}\n"
     "{\"index\":1,\"us\":200}\t{\"x\":2}\n"
-    "{\"index\":2,\"us\":300}\t{\"x\":3}\n";
+    "#head\t00000000000000000300\n"
+    "{\"index\":2,\"us\":400}\t{\"x\":3}\n"
+    "#head\t00000000000000000500\n";
 
+// clang-format off
 const std::string sherlock_golden_data_chunks[] = {
-    "{\"index\":0,\"u",
-    "s\":100}\t{\"x\":1}\r",
-    "\n{\"index\":1,\"us\":200}\t{\"x\":2}\n\r\n{\"index\":2,\"us\":300}\t{\"x",
-    "\":3}\n"};
+  "{\"index\":0,\"u","s\":100}\t{\"x\":1}\r",
+  "\n{\"index\":1,\"us\":200}\t{\"x\":2}\n\r\n"
```

Comment:
Ah, right. As for "\r\n", yes, it's Windows-only, but one of the purposes of this test was to make sure that the remote subscriber parses entries, split by a single '\n', as well as split by any combination of '\r' and '\n'. (see #622)